.semaphore/semaphore.yml

```diff
@@ -4,33 +4,27 @@ agent:
   machine:
     type: e1-standard-2
     os_image: ubuntu1804
-
+global_job_config:
+  prologue:
+    commands:
+      - sem-version python 3.8
+      - sh ~/get-sam-cli.sh
```

Comment:
I don't like it either, but it's their project, and we are not installing it like they recommend leading to lots of packaging conflicts. IMHO we either install it as they recommend or we use something else, otherwise, we are asking for trouble.

Reply:
My point @silvexis, is that we should add it to the makefile or have a script that we can use locally.

Reply:
I don't think it's reasonable to move away from SAM for this reason, or do we plan to move away from AWS CLI?